modelci/app/v1/endpoints/model.py

```diff
@@ -33,3 +33,33 @@ def get_all_model(name: str = None, framework: Framework_ = None, engine: Engine
 def get_model(*, id: str):  # noqa
     model = ModelService.get_model_by_id(id)
     return ModelDetailOut.from_bo(model)
+
+
+@router.get('/structure/{id}', response_model=...)
```

Comment:
Maybe  `response_model=...` should be removed, for fastapi failed to initialize

```
Traceback (most recent call last):
  File "pydantic/validators.py", line 615, in pydantic.validators.find_validators
TypeError: issubclass() arg 1 must be a class

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "~/miniconda3/envs/modelci/lib/python3.7/site-packages/fastapi/utils.py", line 65, in create_response_field
    return response_field(field_info=field_info)
  File "pydantic/fields.py", line 284, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 362, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 538, in pydantic.fields.ModelField.populate_validators
  File "pydantic/validators.py", line 624, in find_validators
RuntimeError: error checking inheritance of Ellipsis (type: ellipsis)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "~/github/NTU/ML-Model-CI/modelci/app/main.py", line 13, in <module>
    from modelci.app.v1.api import api_router
  File "~/github/NTU/ML-Model-CI/modelci/app/v1/api.py", line 10, in <module>
    from modelci.app.v1.endpoints import model
  File "~/github/NTU/ML-Model-CI/modelci/app/v1/endpoints/model.py", line 39, in <module>
    async def get_model_structure(id: str):  # noqa
  File "~/miniconda3/envs/modelci/lib/python3.7/site-packages/fastapi/routing.py", line 517, in decorator
    callbacks=callbacks,
  File "~/miniconda3/envs/modelci/lib/python3.7/site-packages/fastapi/routing.py", line 463, in add_api_route
    callbacks=callbacks,
  File "~/miniconda3/envs/modelci/lib/python3.7/site-packages/fastapi/routing.py", line 304, in __init__
    name=response_name, type_=self.response_model
  File "~/miniconda3/envs/modelci/lib/python3.7/site-packages/fastapi/utils.py", line 68, in create_response_field
    f"Invalid args for response field! Hint: check that {type_} is a valid pydantic field type"
fastapi.exceptions.FastAPIError: Invalid args for response field! Hint: check that Ellipsis is a valid pydantic field type

Process finished with exit code 1
```

Reply:
Commented this line to prevent breaking changes